### PR TITLE
feat: new search index and auto scraping in CI

### PR DESCRIPTION
**Pull request by FabianLars**

### CODE REVIEW

File: src/theme/SearchBar/style.css [hunk 1]
```diff
@@ -72,15 +96,22 @@
...
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
+}
+
+.meilisearch-autocomplete .docs-searchbar-suggestion--subcategory-column {
+  width: 30%;
+  text-align: right;
+  position: relative;
+  padding: 5.3333333333px 10.6666666667px;
```
Comment (lorenzolewis) [inline src/theme/SearchBar/style.css:105]: That's some very precise padding

Comment (FabianLars) [reply src/theme/SearchBar/style.css:105]: yeah, i just copy pasted it hoping it would work in our custom-ish css too 😂

**Comment by FabianLars** [comment]
@lorenzolewis any last concerns?

**Comment by lorenzolewis** [comment]
Did you want to update the netlify action along with this PR @FabianLars ?

**Comment by FabianLars** [comment]
i thought i just did 😂

**Comment by FabianLars** [comment]
okay now i was smart enough to actually click on save 💪 thanks for the reminder

**Comment by lorenzolewis** [comment]
Is there an _easy_ way to also include some text from the page here as well? If the results aren't matching against anything in the body and only the header then even just a snippet of the first "content" in the page could be useful context in the results. 

Screenshot 2023-01-27 at 16 10 01

Here's an example of what I mean (and maybe truncating these to a max of 2/3 lines would be a good shout):

Screenshot 2023-01-27 at 16 12 03

**Comment by FabianLars** [comment]
About the first thing: Yes i want to look into it, i _think_ the codeblock prevents it from finding the text = i think it can only find the text if it's immediately below the heading, but need to do a bit of testing to confirm that.

maybe we can use line-clamp for the second one.

**Comment by lorenzolewis** [comment]
Ah, then this is fine since we'll have to rewrite it later for Astro anyway

**Review by lorenzolewis — APPROVED** [review]

**Comment by FabianLars** [comment]
i was wrong, meilisearch just doesn't return it unless you typed text that is in that paragraph. i'll take a look at the settings, but this shouldn't need any changes to the tauri-docs repo so we can merge it already.

**Pull request merged**